pkgs/applications/office/scribus/unstable.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchsvn, makeWrapper, pkgconfig, cmake, qtbase, cairo, pixman,
+boost, cups, fontconfig, freetype, hunspell, libjpeg, libtiff, libxml2, lcms2,
+podofo, poppler, poppler_data, python2, harfbuzz, qtimageformats, qttools }:
+
+let
+  pythonEnv = python2.withPackages(ps: [ps.tkinter ps.pillow]);
+  revision = "22730";
+in 
+stdenv.mkDerivation rec {
+  name = "scribus-unstable-${version}";
+  version = "2018-10-13";
+
+  src = fetchsvn {
+    url = "svn://scribus.net/trunk/Scribus";
+    rev = revision;
+    sha256 = "1nlg4qva0fach8fi07r1pakjjlijishpwzlgpnxyaz7r31yjaw63";
+  };
+
+  enableParallelBuilding = true;
+
+  buildInputs = [
+    makeWrapper pkgconfig cmake qtbase cairo pixman boost cups fontconfig
+    freetype hunspell libjpeg libtiff libxml2 lcms2 podofo poppler
+    poppler_data python2 pythonEnv harfbuzz qtimageformats qttools
```

Comment:
`python2` should already be in the pythonEnv I think.

Reply:
I fixed it and rebased to master. It starts building, so I'd say it looks good.